### PR TITLE
[READY] Brig phys changes

### PR DESCRIPTION
**Pull request by CoreFlare**
## About The Pull Request

Changes Brig phys so you can't just AFK as it to unlock blueshield.

## Why It's Good For The Game

People are AFKing as brig phys in order to unlock blueshield, which is bad. They should know medical first.

## Changelog
:cl:
tweak: Brig Phys no longer counts towards security hours, it counts towards medical hours.
tweak: You need medical hours in order to unlock Brig Phys now.
/:cl:

### CODE REVIEW

File: modular_skyrat/code/modules/jobs/job_types/brig_physician.dm [hunk 1]
```diff
@@ -10,7 +10,7 @@
 	selection_color = "#c02f2f"
 	minimal_player_age = 7
 	exp_requirements = 120 //SKYRAT CHANGE - lowers security exp requirement
```
Comment (CinderWC) [inline modular_skyrat/code/modules/jobs/job_types/brig_physician.dm:12]: Fix this line to say "medical exp requirement"

Comment (CoreFlare) [reply modular_skyrat/code/modules/jobs/job_types/brig_physician.dm]: addressed in third commit

**Comment by CoreFlare** [comment]
sbeedomergo blease

**Comment by DexterDude** [comment]
yes

**Pull request merged**